Dockerfile.dev

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:xenial
+
+# gcc for cgo
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		wget \
+		pkg-config \
+		git \
+		curl \
+		ca-certificates \
+		libseccomp-dev \
+		parallel \
+		uidmap \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.11.9
```

Comment:
why 1.11 ?

Reply:
Also, based on what is set in .travis.yml